cli/cmd/encore/logs.go

```diff
@@ -92,6 +93,10 @@ func streamLogs(appRoot, envName string) {
 	// Use the same configuration as the runtime
 	zerolog.TimeFieldFormat = time.RFC3339Nano
 
+	if !logsQuiet {
+		os.Stdout.WriteString("Connected, waiting for logs...")
```

Comment:
What do you think of maybe using `aurora` to gray scale this log line? (so it's not as visible as the logs when they do start coming in?)

See https://github.com/encoredev/encore/blob/main/cli/cmd/encore/run.go#L73 for an example

Reply:
Alternatively we could make it a spinner that we delete when logs show up

Reply:
@eandre I went for a message to mirror what the web app does. Do you feel strongly about the spinner?

Reply:
Don't feel strongly at all, ship it :)